modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
```

Comment:
Suggestion: Connecting to a virtual machine via SSH

Reply:
Leaving the title as-is so it doesn't get confused with the "Accessing a virtual machine instance via SSH" module.